glean-core/android/src/main/java/mozilla/telemetry/glean/config/Configuration.kt

```diff
@@ -62,59 +62,23 @@ internal class FfiConfiguration(
  * @property serverEndpoint the server pings are sent to. Please note that this is
  *           is only meant to be changed for tests.
  * @property maxEvents the number of events to store before the events ping is sent
- * @property logPings whether to log ping contents to the console. This is only meant to be used
- *           internally by the `GleanDebugActivity`.
  * @property httpClient The HTTP client implementation to use for uploading pings.
  * @property channel the release channel the application is on, if known. This will be
  *           sent along with all the pings, in the `client_info` section.
  */
-data class Configuration internal constructor(
-    val serverEndpoint: String,
+data class Configuration @JvmOverloads internal constructor(
```

Comment:
Does the `internal` here (since the public one is being removed) still allow access for the consuming app to create a Configuration object?

Reply:
This is a great question and shows my lack of experience with Kotlin. I actually don't know, but makes sense that making it internal would not allow access to consuming apps. I will verify, thanks for noticing.

Reply:
No, it doesn't - this should be removed as suggested in my review :)